src/main/java/org/example/daos/ProjectDao.java

```diff
@@ -121,4 +123,46 @@ public void addEmplyee(final List<Employee> employeeList,
         }
 
     }
+
+    public List<ProjectResponse> getProjects() throws SQLException {
```

Comment:
getAllProjects() would prevent a mistake in future, getProjects() and getProject() function names are very similar